internal/validation/validate_externalauth.go

```diff
@@ -181,12 +181,11 @@ var (
 func validateTokenIssuerProfile(ctx context.Context, op operation.Operation, fldPath *field.Path, newObj, oldObj *api.TokenIssuerProfile) field.ErrorList {
 	errs := field.ErrorList{}
 
-	// TODO why didn't we make this a URL?
```

Comment:
Answer: To do so would have presumed the string was a syntactically valid URL before validation could check it.  Also, conversion from external to internal resource models is supposed to never fail.